src/JsonSchema/Constraints/UndefinedConstraint.php

```diff
@@ -169,7 +169,8 @@ protected function validateCommonProperties(&$value, $schema = null, JsonPointer
             } elseif (isset($schema->required) && !is_array($schema->required)) {
                 // Draft 3 - Required attribute - e.g. "foo": {"type": "string", "required": true}
                 if ($schema->required && $value instanceof self) {
-                    $this->addError($path, 'Is missing and it is required', 'required');
+                    $propertyName = current($path->getPropertyPaths());
+                    $this->addError($path, "The property " . $propertyName . " is required", 'required');
```

Comment:
Per my previous comments, please alter this line to use the new error handler. The syntax you are currently using is deprecated and from 6.0.0 is no longer supported.